### PR TITLE
feat: Add Categories section into the project

### PR DESCRIPTION
**Pull request by HjsCS**

### CODE REVIEW

File: src/components/Categories.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+            {/* instrumental category */}
+            <div className="Categories--card Categories--cards-instrumentalcard">
+                <InstrumentalIcon />
+                <div className="Categories--cards-content Categories--cards-instrumentalcard-content">
+                    <h3 className="font_h3_headline dark-colored-text">Instrumental</h3>
+                    <p className="font_main_regular dark-colored-text">Duis nec sodales nibh. Proin lectus tortor, rutrum vel vulputate vitae</p>
+                </div>
+            </div>
+
```
Comment (Howard-Cui) [inline src/components/Categories.tsx:44]: The font colour should be light rather than dark

**Review by Howard-Cui — COMMENTED** [review]
The font colour of the instrument card should be dark

File: src/components/Categories.tsx (hunk 1 above)
Comment (HjsCS) [reply src/components/Categories.tsx]: right!!!

**Review by Howard-Cui — APPROVED** [review]
LGTM

**Comment by Prom1s1ngYoung** [comment]
LGTM

**Pull request merged**